### DISCUSSION TITLE
Ability to play audio without adding MediaElement to the UI

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [CommunityToolkit/Maui#951] [New Feature Discussions]
In the .NET Podcast app I have to insert a MediaElement into the UI, even though I really just want a  global audio player. It shouldn't need a UI to play audio.

I am wondering if this is a new exposed class perhaps that isn't a UI element?

**Comment by pictos** [comment]
@jamesmontemagno just wondering, if you use the [`MediaManager`](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.shared.cs) directly, are you able to play an audio without the UI? This Manager creates and manage the player for each platform

**Reply by jamesmontemagno** [reply]
I was not able to :( I tried it in the podcast app and nothing played back. However, when I added it into the UI it seemed to work... i'll try again though, but worth a try as this I think will be a common ask.

**Reply by bijington** [reply]
This might feel like a non solution at least in terms of the toolkit but there is this handy package that requires no UI for audio playback https://github.com/jfversluis/Plugin.Maui.Audio. I think we might need to check on streaming support though as I don't recall where we got to with that

**Reply by jamesmontemagno** [reply]
@bijington  yup, I had talked to @jfversluis about this and I am not sure it has streaming support currently.

I sort of feel like audio & video are different... event though they have similar functionality, but audio has deeper integrations into the OS

**Comment by Kapusch** [comment]
I actually had the same requirement and finally got to add the MediaElement to the View to make it work.
As suggested above by Shaun, the plugin from Gerald supports it, but I feel like the MediaElement is really much more stable, at least on iOS.

**Reply by bijington** [reply]
> I actually had the same requirement and finally got to add the MediaElement to the View to make it work.
> 
> As suggested above by Shaun, the plugin from Gerald supports it, but I feel like the MediaElement is really much more stable, at least on iOS.

Out of interest what do you feel is unstable about Plugin.Maui.Audio?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
